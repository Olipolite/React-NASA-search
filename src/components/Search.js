import React, { useState } from "react";
import "../styles/search.css";
import getImages from "../requests/getImages";

function Search() {
  const [value, setValue] = useState();

  return (
    <>
    <form className="search-form">
    <input
     className="search-input"
     type="text"
     onChange={(e) => setValue(e.target.value)}
    />
     <button className="button" type="submit">Search</button>
    </form>
    </>
  );  
};

export default Search;