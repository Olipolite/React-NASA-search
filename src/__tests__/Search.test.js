import { render } from '@testing-library/react';
import Search from '../components/Search';

describe("Search", () => {
  const setSearchResults = jest.fn();

    it("renders correctly", () => {
      const { asFragment } = render(<Search setSearchResults={setSearchResults} />);
  
      expect(asFragment()).toMatchSnapshot();
    });
  });