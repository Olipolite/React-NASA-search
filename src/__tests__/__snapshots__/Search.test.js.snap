// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<DocumentFragment>
  <li>
    <input
      class="search-input"
      type="text"
    />
    <button
      class="button"
      type="submit"
    >
      Submit
    </button>
  </li>
</DocumentFragment>
`;
